lib/main.coffee

```diff
@@ -86,19 +86,25 @@ module.exports =
       type: 'boolean'
       default: true
       order: 13
+    spaceTriggersAutocomplete:
```

Comment:
I dont think there should be an option for it. I think the provider should decide. For example an objective c provider might want to trigger after `[self |`, but the fuzzy provider should never trigger after space.